pkg/version/check_test.go

```diff
@@ -18,26 +18,19 @@ import (
 	"fmt"
 	"net/http"
 	"net/url"
+	"os"
 	"testing"
 	"time"
 
 	"github.com/coreos/go-semver/semver"
-	"github.com/pingcap/check"
 	"github.com/pingcap/kvproto/pkg/metapb"
 	"github.com/pingcap/ticdc/cdc/model"
-	"github.com/pingcap/ticdc/pkg/util/testleak"
+	"github.com/stretchr/testify/require"
+
```

Comment:
```suggestion
	"github.com/stretchr/testify/require"
```

Reply:
??? what's the difference?

Reply:
@overvenus He should have meant the empty line below.

Reply:
oo

Reply:
so, it's a problem adding an empty line here?

Reply:
See: https://github.com/pingcap/ticdc/pull/2920#discussion_r716200957

Reply:
removed